feat: Config service configuration (#682)
safe-global/safe-docs: pages/config-service-configuration/overview.mdx

# Safe Config Service Django configuration

> This document describes how to configure the Config Service through the Django admin panel in a safe way.

Reviewer: ```suggestion
This document describes how to configure the Config Service through the Django admin panel correctly.
```

## Overview

> The Config Service is a Django application that allows you to manage configurations for your services. It provides a web interface where you can create, update, and delete configurations, as well as manage groups of permissions to control who can access the configurations.

Reviewer: ```suggestion
The Config Service is a Django application that allows you to manage configurations for your clients. It provides a web interface where you can create, update, and delete configurations, as well as manage groups of permissions to control who can access the configurations.
```
